enforce_test.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
I don't think we need to test the client alone. We should use integration test to test Casbin-Server and this client together, e.g., via Docker and gRPC in Travis CI.